examples/step-31/step-31.cc

```diff
@@ -1294,8 +1294,8 @@ namespace Step31
     // the update flags, zeroing out the local arrays and getting the values
     // of the old solution at the quadrature points. Then we are ready to loop
     // over the quadrature points on the cell.
-    auto cell             = stokes_dof_handler.begin_active(),
-         endc             = stokes_dof_handler.end();
+    auto cell             = stokes_dof_handler.begin_active();
+    auto endc             = stokes_dof_handler.end();
```

Comment:
Should the end cells not be `const auto`?

Reply:
Indeed, even though I liked the symmetry in the three lines.

Reply:
Sorry about that :-D